Modelica/Resources/Scripts/Dymola/Fluid/TanksWithOverflow/plot level and port.m_flow.mos

```diff
@@ -64,8 +64,7 @@ createPlot(id = 1,
  bottomTitleType = 1);
 createPlot(id = 1,
  position = {3, 3, 771, 270},
- y = {"upperTank.ports[1].m_flow", "pipe.port_a.m_flow", "overflow.port_a.m_flow",
- "tank3.ports[2].m_flow"},
+ y = {"upperTank.ports[1].m_flow", "pipe.port_a.m_flow", "overflow.port_a.m_flow"},
```

Comment:
Why not `lowerTank.ports[2].m_flow`?

Reply:
I experimented with that - but it's the same as `overflow.port_a.m_flow`; so it seemed redundant.

Reply:
And easier to understand what overflow is.